compute/instances/create-manage-windows-instances/create_windows_server_external_ip.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START compute_create_windows_instance_external_ip]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+	"google.golang.org/protobuf/proto"
+)
+
+// createWndowsServerInstanceExternalIP creates a new Windows Server instance that has an external IP address.
+func createWndowsServerInstanceExternalIP(w io.Writer, projectID, zone, instanceName, machineType, sourceImageFamily string) error {
+	// projectID := "your_project_id"
+	// zone := "europe-central2-b"
+	// instanceName := "your_instance_name"
+	// machineType := "n1-standard-1"
+	// sourceImageFamily := "windows-2012-r2"
+
+	ctx := context.Background()
+	instancesClient, err := compute.NewInstancesRESTClient(ctx)
+	if err != nil {
+		return fmt.Errorf("NewInstancesRESTClient: %v", err)
+	}
+	defer instancesClient.Close()
+
+	req := &computepb.InsertInstanceRequest{
+		Project: projectID,
+		Zone:    zone,
+		InstanceResource: &computepb.Instance{
```

Comment:
The line lengths on this request type are going to require lots of horizontal scrolling.

What if we pulled the `&computepb.Instance{}` into it's own variable to help with this?

``` golang
inst := &computepb.Instance{ /* ... */ }
req := &computepb.InsertInstanceRequest{
    // ...
    InstanceResource: inst,
}
```